src/lib/libcore.js

```diff
@@ -320,6 +322,7 @@ addToLibrary({
 #endif // ALLOW_MEMORY_GROWTH
   },
 
+#if !GROWABLE_ARRAYBUFFERS
```

Comment:
This function is still being used here without a guard for `!GROWABLE_ARRAYBUFFERS`:
https://github.com/emscripten-core/emscripten/blob/a3cf37123035461427d06dfc4372cded984baa1b/src/lib/libcore.js#L2422-L2427

(i.e. that `#if` should be adjusted to `if PURE_WASI && !GROWABLE_ARRAYBUFFERS`)